recipes/xz/xz.inc

```diff
@@ -0,0 +1,49 @@
+DESCRIPTION = "XZ Utils is free general-purpose data compression software with a high compression ratio, the successor to LZMA Utils"
+HOMEPAGE = "http://tukaani.org/xz/"
+
+COMPATIBLE_HOST_ARCHS = ".*linux"
+RECIPE_TYPES = "machine native"
+
+# The COPYING file starts
+#
+#   Different licenses apply to different files in this package. Here
+#   is a rough summary of which licenses apply to which parts of this
+#   package (but check the individual files to be sure!):
+#
+# Boiling that down to a single license tag is not easy. Most files
+# are public domain, the remaining are under some GPL variant.
+
+LICENSE = "GPLv2"
```

Comment:
Hm, not sure. LGPL-2.1 is only if the bundled getopt_long implementation gets used, which it won't as long as we're using glibc. The mention of GPL-3 is followed by "None of these files end up in the binaries being built.". That leaves "public domain" for the main binaries, and "GPLv2+" for the *grep, *diff, *less wrappers. I see we have monstrosities like "GPLv2 & (LGPLv2.1 | MPL-1.1 | BSD)" - what does that mean, and how do we properly handle recipes with different licenses for different packages?